[2022-08-05 15:11:55.826] DEBUG: database.__call__ @ 43: record=Notification(12, jlondon@robertsonhayles.com, [NotificationField(description='File number', value='PRO 4852/2022'), NotificationField(description='Title', value='In the Estate of Jane Gray Fraser GORDON deceased'), NotificationField(description='Party', value='Norman Mclean Gordon')])
[2022-08-05 15:12:08.8] DEBUG: database.__call__ @ 56: Sent
[2022-08-05 15:12:26.725] DEBUG: routes.cancel_registration @ 70: token value=None
[2022-08-05 15:12:27.85] DEBUG: routes.cancel_registration @ 70: token value=None
[2022-08-05 15:12:37.92] INFO: update.update @ 254: REN
[2022-08-05 15:12:37.94] INFO: update.update @ 258: 29
[2022-08-05 15:13:04.119] INFO: update.update @ 254: STAT
[2022-08-05 15:13:04.120] INFO: update.update @ 258: 0
[2022-08-05 15:13:29.232] INFO: update.get_skipped_multipage_parties @ 404: Getting skipped multipage matters
[2022-08-05 15:13:29.260] INFO: update.rescrape @ 484: rescraping
[2022-08-05 15:13:29.333] DEBUG: update.db @ 120: Closing db
[2022-08-05 15:13:29.334] DEBUG: update.db @ 122: Closed db
[2022-08-05 15:13:29.337] INFO: update.update_db @ 87: Sleeping for 1800 seconds
[2022-08-05 15:43:31.743] INFO: update.update @ 249: 2022
[2022-08-05 15:43:31.743] INFO: update.update @ 254: CAV
[2022-08-05 15:43:31.743] DEBUG: update.db @ 112: Loading db
[2022-08-05 15:43:31.745] INFO: update.update @ 258: 80
[2022-08-05 15:43:31.745] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-05 15:44:32.849] DEBUG: update.browser @ 149: RoboBrower ready for searching
[2022-08-05 15:44:57.411] INFO: update.update @ 254: CIT
[2022-08-05 15:44:57.412] INFO: update.update @ 258: 15
[2022-08-05 15:45:21.101] INFO: update.update @ 254: ELEC
[2022-08-05 15:45:21.109] INFO: update.update @ 258: 6
[2022-08-05 15:45:42.911] INFO: update.update @ 254: PRO
[2022-08-05 15:45:42.918] INFO: update.update @ 258: 4853
[2022-08-05 15:46:06.168] INFO: update.update @ 254: REN
[2022-08-05 15:46:06.168] INFO: update.update @ 258: 29
[2022-08-05 15:46:28.12] INFO: update.update @ 254: STAT
[2022-08-05 15:46:28.13] INFO: update.update @ 258: 0
[2022-08-05 15:46:50.911] INFO: update.get_skipped_multipage_parties @ 404: Getting skipped multipage matters
[2022-08-05 15:46:50.934] INFO: update.rescrape @ 484: rescraping
[2022-08-05 15:46:50.957] DEBUG: update.db @ 120: Closing db
[2022-08-05 15:46:50.957] DEBUG: update.db @ 122: Closed db
[2022-08-05 15:46:51.10] INFO: update.update_db @ 87: Sleeping for 1800 seconds
[2022-08-05 16:16:53.182] INFO: update.update @ 249: 2022
[2022-08-05 16:16:53.183] INFO: update.update @ 254: CAV
[2022-08-05 16:16:53.183] DEBUG: update.db @ 112: Loading db
[2022-08-05 16:16:53.184] INFO: update.update @ 258: 81
[2022-08-05 16:16:53.185] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-05 16:17:22.824] DEBUG: update.browser @ 149: RoboBrower ready for searching
[2022-08-05 16:17:48.137] INFO: update.update @ 254: CIT
[2022-08-05 16:17:48.138] INFO: update.update @ 258: 15
[2022-08-05 16:18:12.637] INFO: update.update @ 254: ELEC
[2022-08-05 16:18:12.639] INFO: update.update @ 258: 6
[2022-08-05 16:18:38.403] INFO: update.update @ 254: PRO
[2022-08-05 16:18:38.408] INFO: update.update @ 258: 4855
[2022-08-05 16:19:03.920] INFO: update.update @ 254: REN
[2022-08-05 16:19:03.921] INFO: update.update @ 258: 29
[2022-08-05 16:19:28.878] INFO: update.update @ 254: STAT
[2022-08-05 16:19:28.880] INFO: update.update @ 258: 0
[2022-08-05 16:19:55.114] INFO: update.get_skipped_multipage_parties @ 404: Getting skipped multipage matters
[2022-08-05 16:19:55.146] INFO: update.rescrape @ 484: rescraping
[2022-08-05 16:19:55.208] DEBUG: update.db @ 120: Closing db
[2022-08-05 16:19:55.208] DEBUG: update.db @ 122: Closed db
[2022-08-05 16:19:55.210] INFO: update.update_db @ 87: Sleeping for 1800 seconds
[2022-08-05 16:49:57.394] INFO: update.update @ 249: 2022
[2022-08-05 16:49:57.394] INFO: update.update @ 254: CAV
[2022-08-05 16:49:57.394] DEBUG: update.db @ 112: Loading db
[2022-08-05 16:49:57.396] INFO: update.update @ 258: 81
[2022-08-05 16:49:57.396] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-05 16:50:34.113] DEBUG: update.browser @ 149: RoboBrower ready for searching
[2022-08-05 16:50:52.895] INFO: update.update @ 254: CIT
[2022-08-05 16:50:52.895] INFO: update.update @ 258: 15
[2022-08-05 16:51:13.810] INFO: update.update @ 254: ELEC
[2022-08-05 16:51:13.811] INFO: update.update @ 258: 6
[2022-08-05 16:51:32.435] INFO: update.update @ 254: PRO
[2022-08-05 16:51:32.443] INFO: update.update @ 258: 4855
[2022-08-05 16:51:52.283] INFO: update.update @ 254: REN
[2022-08-05 16:51:52.283] INFO: update.update @ 258: 29
[2022-08-05 16:52:11.654] INFO: update.update @ 254: STAT
[2022-08-05 16:52:11.655] INFO: update.update @ 258: 0
[2022-08-05 16:52:30.116] INFO: update.get_skipped_multipage_parties @ 404: Getting skipped multipage matters
[2022-08-05 16:52:30.151] INFO: update.rescrape @ 484: rescraping
[2022-08-05 16:52:30.173] DEBUG: update.db @ 120: Closing db
[2022-08-05 16:52:30.173] DEBUG: update.db @ 122: Closed db
[2022-08-05 16:52:30.176] INFO: update.update_db @ 87: Sleeping for 1800 seconds
[2022-08-05 17:22:32.561] INFO: update.update @ 249: 2022
[2022-08-05 17:22:32.562] INFO: update.update @ 254: CAV
[2022-08-05 17:22:32.562] DEBUG: update.db @ 112: Loading db
[2022-08-05 17:22:32.564] INFO: update.update @ 258: 81
[2022-08-05 17:22:32.564] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-05 17:22:39.974] DEBUG: update.browser @ 149: RoboBrower ready for searching
[2022-08-05 17:22:58.739] INFO: update.update @ 254: CIT
[2022-08-05 17:22:58.740] INFO: update.update @ 258: 15
[2022-08-05 17:23:17.442] INFO: update.update @ 254: ELEC
[2022-08-05 17:23:17.446] INFO: update.update @ 258: 6
[2022-08-05 17:23:36.174] INFO: update.update @ 254: PRO
[2022-08-05 17:23:36.180] INFO: update.update @ 258: 4855
[2022-08-05 17:23:54.185] INFO: update.update @ 254: REN
[2022-08-05 17:23:54.186] INFO: update.update @ 258: 29
[2022-08-05 17:24:13.237] INFO: update.update @ 254: STAT
[2022-08-05 17:24:13.238] INFO: update.update @ 258: 0
[2022-08-05 17:24:31.328] INFO: update.get_skipped_multipage_parties @ 404: Getting skipped multipage matters
[2022-08-05 17:24:31.364] INFO: update.rescrape @ 484: rescraping
[2022-08-05 17:24:31.432] DEBUG: update.db @ 120: Closing db
[2022-08-05 17:24:31.432] DEBUG: update.db @ 122: Closed db
[2022-08-05 17:24:31.436] INFO: update.update_db @ 87: Sleeping for 1800 seconds
[2022-08-05 17:54:33.722] INFO: update.update @ 249: 2022
[2022-08-05 17:54:33.722] INFO: update.update @ 254: CAV
[2022-08-05 17:54:33.723] DEBUG: update.db @ 112: Loading db
[2022-08-05 17:54:33.724] INFO: update.update @ 258: 81
[2022-08-05 17:54:33.724] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-05 17:54:40.994] DEBUG: update.browser @ 149: RoboBrower ready for searching
[2022-08-05 17:55:00.629] INFO: update.update @ 254: CIT
[2022-08-05 17:55:00.630] INFO: update.update @ 258: 15
[2022-08-05 17:55:19.245] INFO: update.update @ 254: ELEC
[2022-08-05 17:55:19.246] INFO: update.update @ 258: 6
[2022-08-05 17:55:38.911] INFO: update.update @ 254: PRO
[2022-08-05 17:55:38.940] INFO: update.update @ 258: 4855
[2022-08-05 17:56:02.130] INFO: update.update @ 254: REN
[2022-08-05 17:56:02.131] INFO: update.update @ 258: 29
[2022-08-05 17:56:24.214] INFO: update.update @ 254: STAT
[2022-08-05 17:56:24.221] INFO: update.update @ 258: 0
[2022-08-05 17:56:43.775] INFO: update.get_skipped_multipage_parties @ 404: Getting skipped multipage matters
[2022-08-05 17:56:43.825] INFO: update.rescrape @ 484: rescraping
[2022-08-05 17:56:43.894] DEBUG: update.db @ 120: Closing db
[2022-08-05 17:56:43.895] DEBUG: update.db @ 122: Closed db
[2022-08-05 17:56:43.903] INFO: update.update_db @ 87: Sleeping for 1800 seconds
[2022-08-05 18:26:46.194] DEBUG: update.db @ 120: Closing db
[2022-08-05 18:26:46.194] DEBUG: update.db @ 122: Closed db
[2022-08-05 18:26:46.194] INFO: update.update_db @ 87: Sleeping for 221593 seconds
[2022-08-06 04:47:07.414] INFO: __init__.create_app @ 30: App startup in production mode
[2022-08-06 04:47:07.416] INFO: update.update_public_holidays @ 215: Updating public holidays
[2022-08-06 04:47:07.417] DEBUG: update.db @ 112: Loading db
[2022-08-06 04:47:08.747] DEBUG: update.update_public_holidays @ 228: Updated public holidays
[2022-08-06 04:47:08.747] DEBUG: update.public_holidays @ 196: Loading public holidays
[2022-08-06 04:47:08.750] DEBUG: update.db @ 120: Closing db
[2022-08-06 04:47:08.751] DEBUG: update.db @ 122: Closed db
[2022-08-06 04:47:08.751] INFO: update.update_db @ 87: Sleeping for 184372 seconds
[2022-08-06 05:47:01.806] INFO: __init__.create_app @ 30: App startup in production mode
[2022-08-06 05:47:01.815] INFO: update.update_public_holidays @ 215: Updating public holidays
[2022-08-06 05:47:01.816] DEBUG: update.db @ 112: Loading db
[2022-08-06 05:47:03.172] DEBUG: update.update_public_holidays @ 228: Updated public holidays
[2022-08-06 05:47:03.172] DEBUG: update.public_holidays @ 196: Loading public holidays
[2022-08-06 05:47:03.176] DEBUG: update.db @ 120: Closing db
[2022-08-06 05:47:03.176] DEBUG: update.db @ 122: Closed db
[2022-08-06 05:47:03.177] INFO: update.update_db @ 87: Sleeping for 180778 seconds
[2022-08-07 06:32:15.732] INFO: __init__.create_app @ 30: App startup in production mode
[2022-08-07 06:32:15.734] INFO: update.update_public_holidays @ 215: Updating public holidays
[2022-08-07 06:32:15.734] DEBUG: update.db @ 112: Loading db
[2022-08-07 06:32:17.822] DEBUG: update.update_public_holidays @ 228: Updated public holidays
[2022-08-07 06:32:17.822] DEBUG: update.public_holidays @ 196: Loading public holidays
[2022-08-07 06:32:17.826] DEBUG: update.db @ 120: Closing db
[2022-08-07 06:32:17.827] DEBUG: update.db @ 122: Closed db
[2022-08-07 06:32:17.828] INFO: update.update_db @ 87: Sleeping for 91664 seconds
[2022-08-08 02:47:25.851] INFO: __init__.create_app @ 30: App startup in production mode
[2022-08-08 02:47:25.854] INFO: update.update_public_holidays @ 215: Updating public holidays
[2022-08-08 02:47:25.854] DEBUG: update.db @ 112: Loading db
[2022-08-08 02:47:27.214] DEBUG: update.update_public_holidays @ 228: Updated public holidays
[2022-08-08 02:47:27.214] DEBUG: update.public_holidays @ 196: Loading public holidays
[2022-08-08 02:47:27.217] DEBUG: update.db @ 120: Closing db
[2022-08-08 02:47:27.218] DEBUG: update.db @ 122: Closed db
[2022-08-08 02:47:27.219] INFO: update.update_db @ 87: Sleeping for 18754 seconds
[2022-08-08 05:34:07.725] INFO: __init__.create_app @ 30: App startup in production mode
[2022-08-08 05:34:07.727] INFO: update.update_public_holidays @ 215: Updating public holidays
[2022-08-08 05:34:07.727] DEBUG: update.db @ 112: Loading db
[2022-08-08 05:34:09.977] DEBUG: update.update_public_holidays @ 228: Updated public holidays
[2022-08-08 05:34:09.978] DEBUG: update.public_holidays @ 196: Loading public holidays
[2022-08-08 05:34:09.981] DEBUG: update.db @ 120: Closing db
[2022-08-08 05:34:09.981] DEBUG: update.db @ 122: Closed db
[2022-08-08 05:34:09.982] INFO: update.update_db @ 87: Sleeping for 8752 seconds
[2022-08-08 08:00:11.2] INFO: update.update @ 249: 2022
[2022-08-08 08:00:11.3] INFO: update.update @ 254: CAV
[2022-08-08 08:00:11.3] DEBUG: update.db @ 112: Loading db
[2022-08-08 08:00:11.6] INFO: update.update @ 258: 81
[2022-08-08 08:00:11.6] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 08:00:19.60] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf9ff0a0>
[2022-08-08 08:00:25.280] DEBUG: update.db @ 120: Closing db
[2022-08-08 08:00:25.280] DEBUG: update.db @ 122: Closed db
[2022-08-08 08:00:25.281] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 08:15:26.176] INFO: update.update @ 249: 2022
[2022-08-08 08:15:26.176] INFO: update.update @ 254: CAV
[2022-08-08 08:15:26.177] DEBUG: update.db @ 112: Loading db
[2022-08-08 08:15:26.178] INFO: update.update @ 258: 81
[2022-08-08 08:15:26.181] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 08:15:33.84] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf761af0>
[2022-08-08 08:15:39.713] DEBUG: update.db @ 120: Closing db
[2022-08-08 08:15:39.714] DEBUG: update.db @ 122: Closed db
[2022-08-08 08:15:39.714] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 08:30:40.707] INFO: update.update @ 249: 2022
[2022-08-08 08:30:40.707] INFO: update.update @ 254: CAV
[2022-08-08 08:30:40.708] DEBUG: update.db @ 112: Loading db
[2022-08-08 08:30:40.712] INFO: update.update @ 258: 81
[2022-08-08 08:30:40.713] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 08:30:47.76] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf5af4c0>
[2022-08-08 08:30:54.172] DEBUG: update.db @ 120: Closing db
[2022-08-08 08:30:54.172] DEBUG: update.db @ 122: Closed db
[2022-08-08 08:30:54.172] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 08:45:55.124] INFO: update.update @ 249: 2022
[2022-08-08 08:45:55.125] INFO: update.update @ 254: CAV
[2022-08-08 08:45:55.125] DEBUG: update.db @ 112: Loading db
[2022-08-08 08:45:55.126] INFO: update.update @ 258: 81
[2022-08-08 08:45:55.127] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 08:46:03.281] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf424280>
[2022-08-08 08:46:10.314] DEBUG: update.db @ 120: Closing db
[2022-08-08 08:46:10.315] DEBUG: update.db @ 122: Closed db
[2022-08-08 08:46:10.315] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 09:01:11.255] INFO: update.update @ 249: 2022
[2022-08-08 09:01:11.255] INFO: update.update @ 254: CAV
[2022-08-08 09:01:11.256] DEBUG: update.db @ 112: Loading db
[2022-08-08 09:01:11.257] INFO: update.update @ 258: 81
[2022-08-08 09:01:11.257] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 09:01:18.566] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf222a60>
[2022-08-08 09:01:26.198] DEBUG: update.db @ 120: Closing db
[2022-08-08 09:01:26.198] DEBUG: update.db @ 122: Closed db
[2022-08-08 09:01:26.199] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 09:16:27.189] INFO: update.update @ 249: 2022
[2022-08-08 09:16:27.190] INFO: update.update @ 254: CAV
[2022-08-08 09:16:27.190] DEBUG: update.db @ 112: Loading db
[2022-08-08 09:16:27.192] INFO: update.update @ 258: 81
[2022-08-08 09:16:27.192] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 09:16:33.681] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf068430>
[2022-08-08 09:16:39.916] DEBUG: update.db @ 120: Closing db
[2022-08-08 09:16:39.917] DEBUG: update.db @ 122: Closed db
[2022-08-08 09:16:39.917] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 09:31:40.877] INFO: update.update @ 249: 2022
[2022-08-08 09:31:40.877] INFO: update.update @ 254: CAV
[2022-08-08 09:31:40.878] DEBUG: update.db @ 112: Loading db
[2022-08-08 09:31:40.879] INFO: update.update @ 258: 81
[2022-08-08 09:31:40.879] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 09:31:53.508] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaeea2ca0>
[2022-08-08 09:32:00.591] DEBUG: update.db @ 120: Closing db
[2022-08-08 09:32:00.591] DEBUG: update.db @ 122: Closed db
[2022-08-08 09:32:00.592] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 09:47:01.620] INFO: update.update @ 249: 2022
[2022-08-08 09:47:01.620] INFO: update.update @ 254: CAV
[2022-08-08 09:47:01.620] DEBUG: update.db @ 112: Loading db
[2022-08-08 09:47:01.621] INFO: update.update @ 258: 81
[2022-08-08 09:47:01.622] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 09:47:11.746] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdafa721c0>
[2022-08-08 09:47:18.605] DEBUG: update.db @ 120: Closing db
[2022-08-08 09:47:18.605] DEBUG: update.db @ 122: Closed db
[2022-08-08 09:47:18.605] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 10:02:19.528] INFO: update.update @ 249: 2022
[2022-08-08 10:02:19.528] INFO: update.update @ 254: CAV
[2022-08-08 10:02:19.529] DEBUG: update.db @ 112: Loading db
[2022-08-08 10:02:19.547] INFO: update.update @ 258: 81
[2022-08-08 10:02:19.548] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 10:02:35.11] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf0cfc10>
[2022-08-08 10:02:41.236] DEBUG: update.db @ 120: Closing db
[2022-08-08 10:02:41.236] DEBUG: update.db @ 122: Closed db
[2022-08-08 10:02:41.237] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 10:17:42.271] INFO: update.update @ 249: 2022
[2022-08-08 10:17:42.271] INFO: update.update @ 254: CAV
[2022-08-08 10:17:42.271] DEBUG: update.db @ 112: Loading db
[2022-08-08 10:17:42.275] INFO: update.update @ 258: 81
[2022-08-08 10:17:42.276] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 10:17:50.406] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf615490>
[2022-08-08 10:17:57.667] DEBUG: update.db @ 120: Closing db
[2022-08-08 10:17:57.668] DEBUG: update.db @ 122: Closed db
[2022-08-08 10:17:57.668] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 10:32:58.620] INFO: update.update @ 249: 2022
[2022-08-08 10:32:58.620] INFO: update.update @ 254: CAV
[2022-08-08 10:32:58.621] DEBUG: update.db @ 112: Loading db
[2022-08-08 10:32:58.622] INFO: update.update @ 258: 81
[2022-08-08 10:32:58.622] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 10:33:06.222] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf1f8820>
[2022-08-08 10:33:13.197] DEBUG: update.db @ 120: Closing db
[2022-08-08 10:33:13.197] DEBUG: update.db @ 122: Closed db
[2022-08-08 10:33:13.198] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 10:48:14.131] INFO: update.update @ 249: 2022
[2022-08-08 10:48:14.131] INFO: update.update @ 254: CAV
[2022-08-08 10:48:14.132] DEBUG: update.db @ 112: Loading db
[2022-08-08 10:48:14.133] INFO: update.update @ 258: 81
[2022-08-08 10:48:14.133] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 10:48:20.885] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf3ce520>
[2022-08-08 10:48:27.88] DEBUG: update.db @ 120: Closing db
[2022-08-08 10:48:27.89] DEBUG: update.db @ 122: Closed db
[2022-08-08 10:48:27.90] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 11:03:28.26] INFO: update.update @ 249: 2022
[2022-08-08 11:03:28.27] INFO: update.update @ 254: CAV
[2022-08-08 11:03:28.27] DEBUG: update.db @ 112: Loading db
[2022-08-08 11:03:28.29] INFO: update.update @ 258: 81
[2022-08-08 11:03:28.29] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 11:03:36.926] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf6ea5b0>
[2022-08-08 11:03:43.997] DEBUG: update.db @ 120: Closing db
[2022-08-08 11:03:43.997] DEBUG: update.db @ 122: Closed db
[2022-08-08 11:03:43.998] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 11:18:44.986] INFO: update.update @ 249: 2022
[2022-08-08 11:18:44.987] INFO: update.update @ 254: CAV
[2022-08-08 11:18:44.987] DEBUG: update.db @ 112: Loading db
[2022-08-08 11:18:44.989] INFO: update.update @ 258: 81
[2022-08-08 11:18:44.990] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 11:18:53.89] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf32d4c0>
[2022-08-08 11:18:59.894] DEBUG: update.db @ 120: Closing db
[2022-08-08 11:18:59.894] DEBUG: update.db @ 122: Closed db
[2022-08-08 11:18:59.895] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 11:34:00.837] INFO: update.update @ 249: 2022
[2022-08-08 11:34:00.838] INFO: update.update @ 254: CAV
[2022-08-08 11:34:00.838] DEBUG: update.db @ 112: Loading db
[2022-08-08 11:34:00.839] INFO: update.update @ 258: 81
[2022-08-08 11:34:00.841] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 11:34:07.307] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaece5fa0>
[2022-08-08 11:34:14.654] DEBUG: update.db @ 120: Closing db
[2022-08-08 11:34:14.654] DEBUG: update.db @ 122: Closed db
[2022-08-08 11:34:14.655] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 11:49:15.568] INFO: update.update @ 249: 2022
[2022-08-08 11:49:15.569] INFO: update.update @ 254: CAV
[2022-08-08 11:49:15.569] DEBUG: update.db @ 112: Loading db
[2022-08-08 11:49:15.570] INFO: update.update @ 258: 81
[2022-08-08 11:49:15.570] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 11:49:22.213] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdbc081e20>
[2022-08-08 11:49:29.70] DEBUG: update.db @ 120: Closing db
[2022-08-08 11:49:29.70] DEBUG: update.db @ 122: Closed db
[2022-08-08 11:49:29.71] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 12:04:30.8] INFO: update.update @ 249: 2022
[2022-08-08 12:04:30.8] INFO: update.update @ 254: CAV
[2022-08-08 12:04:30.8] DEBUG: update.db @ 112: Loading db
[2022-08-08 12:04:30.10] INFO: update.update @ 258: 81
[2022-08-08 12:04:30.10] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 12:04:37.122] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf38bc10>
[2022-08-08 12:04:43.927] DEBUG: update.db @ 120: Closing db
[2022-08-08 12:04:43.927] DEBUG: update.db @ 122: Closed db
[2022-08-08 12:04:43.928] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 12:19:45.12] INFO: update.update @ 249: 2022
[2022-08-08 12:19:45.17] INFO: update.update @ 254: CAV
[2022-08-08 12:19:45.18] DEBUG: update.db @ 112: Loading db
[2022-08-08 12:19:45.19] INFO: update.update @ 258: 81
[2022-08-08 12:19:45.19] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 12:19:52.41] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf78fe20>
[2022-08-08 12:19:58.246] DEBUG: update.db @ 120: Closing db
[2022-08-08 12:19:58.246] DEBUG: update.db @ 122: Closed db
[2022-08-08 12:19:58.247] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 12:34:59.203] INFO: update.update @ 249: 2022
[2022-08-08 12:34:59.204] INFO: update.update @ 254: CAV
[2022-08-08 12:34:59.204] DEBUG: update.db @ 112: Loading db
[2022-08-08 12:34:59.206] INFO: update.update @ 258: 81
[2022-08-08 12:34:59.206] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 12:35:06.738] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf020790>
[2022-08-08 12:35:12.949] DEBUG: update.db @ 120: Closing db
[2022-08-08 12:35:12.949] DEBUG: update.db @ 122: Closed db
[2022-08-08 12:35:12.950] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 12:50:13.916] INFO: update.update @ 249: 2022
[2022-08-08 12:50:13.916] INFO: update.update @ 254: CAV
[2022-08-08 12:50:13.916] DEBUG: update.db @ 112: Loading db
[2022-08-08 12:50:13.918] INFO: update.update @ 258: 81
[2022-08-08 12:50:13.919] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 12:50:20.969] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf667790>
[2022-08-08 12:50:28.205] DEBUG: update.db @ 120: Closing db
[2022-08-08 12:50:28.206] DEBUG: update.db @ 122: Closed db
[2022-08-08 12:50:28.206] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 13:05:29.118] INFO: update.update @ 249: 2022
[2022-08-08 13:05:29.119] INFO: update.update @ 254: CAV
[2022-08-08 13:05:29.119] DEBUG: update.db @ 112: Loading db
[2022-08-08 13:05:29.121] INFO: update.update @ 258: 81
[2022-08-08 13:05:29.122] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 13:05:36.361] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf28b1c0>
[2022-08-08 13:05:42.463] DEBUG: update.db @ 120: Closing db
[2022-08-08 13:05:42.463] DEBUG: update.db @ 122: Closed db
[2022-08-08 13:05:42.464] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 13:20:43.446] INFO: update.update @ 249: 2022
[2022-08-08 13:20:43.446] INFO: update.update @ 254: CAV
[2022-08-08 13:20:43.447] DEBUG: update.db @ 112: Loading db
[2022-08-08 13:20:43.448] INFO: update.update @ 258: 81
[2022-08-08 13:20:43.448] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 13:20:51.204] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf6ddaf0>
[2022-08-08 13:20:57.513] DEBUG: update.db @ 120: Closing db
[2022-08-08 13:20:57.513] DEBUG: update.db @ 122: Closed db
[2022-08-08 13:20:57.514] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 13:35:58.502] INFO: update.update @ 249: 2022
[2022-08-08 13:35:58.503] INFO: update.update @ 254: CAV
[2022-08-08 13:35:58.503] DEBUG: update.db @ 112: Loading db
[2022-08-08 13:35:58.505] INFO: update.update @ 258: 81
[2022-08-08 13:35:58.506] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 13:36:06.765] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf175af0>
[2022-08-08 13:36:13.537] DEBUG: update.db @ 120: Closing db
[2022-08-08 13:36:13.538] DEBUG: update.db @ 122: Closed db
[2022-08-08 13:36:13.538] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 13:51:14.513] INFO: update.update @ 249: 2022
[2022-08-08 13:51:14.514] INFO: update.update @ 254: CAV
[2022-08-08 13:51:14.514] DEBUG: update.db @ 112: Loading db
[2022-08-08 13:51:14.515] INFO: update.update @ 258: 81
[2022-08-08 13:51:14.516] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 13:51:39.382] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf552250>
[2022-08-08 13:51:46.228] DEBUG: update.db @ 120: Closing db
[2022-08-08 13:51:46.229] DEBUG: update.db @ 122: Closed db
[2022-08-08 13:51:46.230] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 14:06:47.200] INFO: update.update @ 249: 2022
[2022-08-08 14:06:47.200] INFO: update.update @ 254: CAV
[2022-08-08 14:06:47.200] DEBUG: update.db @ 112: Loading db
[2022-08-08 14:06:47.202] INFO: update.update @ 258: 81
[2022-08-08 14:06:47.202] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 14:07:24.77] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf297eb0>
[2022-08-08 14:07:30.979] DEBUG: update.db @ 120: Closing db
[2022-08-08 14:07:30.980] DEBUG: update.db @ 122: Closed db
[2022-08-08 14:07:30.981] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 14:22:32.79] INFO: update.update @ 249: 2022
[2022-08-08 14:22:32.80] INFO: update.update @ 254: CAV
[2022-08-08 14:22:32.80] DEBUG: update.db @ 112: Loading db
[2022-08-08 14:22:32.82] INFO: update.update @ 258: 81
[2022-08-08 14:22:32.83] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 14:22:40.7] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdafaa1e80>
[2022-08-08 14:22:47.198] DEBUG: update.db @ 120: Closing db
[2022-08-08 14:22:47.199] DEBUG: update.db @ 122: Closed db
[2022-08-08 14:22:47.200] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 14:37:48.241] INFO: update.update @ 249: 2022
[2022-08-08 14:37:48.244] INFO: update.update @ 254: CAV
[2022-08-08 14:37:48.245] DEBUG: update.db @ 112: Loading db
[2022-08-08 14:37:48.246] INFO: update.update @ 258: 81
[2022-08-08 14:37:48.246] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 14:37:55.840] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaefbd8e0>
[2022-08-08 14:38:02.59] DEBUG: update.db @ 120: Closing db
[2022-08-08 14:38:02.59] DEBUG: update.db @ 122: Closed db
[2022-08-08 14:38:02.60] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 14:53:02.992] INFO: update.update @ 249: 2022
[2022-08-08 14:53:02.992] INFO: update.update @ 254: CAV
[2022-08-08 14:53:02.993] DEBUG: update.db @ 112: Loading db
[2022-08-08 14:53:02.994] INFO: update.update @ 258: 81
[2022-08-08 14:53:02.994] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 14:53:10.29] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf427280>
[2022-08-08 14:53:16.521] DEBUG: update.db @ 120: Closing db
[2022-08-08 14:53:16.521] DEBUG: update.db @ 122: Closed db
[2022-08-08 14:53:16.522] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 15:08:17.513] INFO: update.update @ 249: 2022
[2022-08-08 15:08:17.513] INFO: update.update @ 254: CAV
[2022-08-08 15:08:17.514] DEBUG: update.db @ 112: Loading db
[2022-08-08 15:08:17.515] INFO: update.update @ 258: 81
[2022-08-08 15:08:17.516] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 15:08:24.277] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf306250>
[2022-08-08 15:08:30.564] DEBUG: update.db @ 120: Closing db
[2022-08-08 15:08:30.564] DEBUG: update.db @ 122: Closed db
[2022-08-08 15:08:30.566] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 15:23:31.667] INFO: update.update @ 249: 2022
[2022-08-08 15:23:31.668] INFO: update.update @ 254: CAV
[2022-08-08 15:23:31.668] DEBUG: update.db @ 112: Loading db
[2022-08-08 15:23:31.677] INFO: update.update @ 258: 81
[2022-08-08 15:23:31.677] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 15:23:39.23] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaeeb8c70>
[2022-08-08 15:23:46.98] DEBUG: update.db @ 120: Closing db
[2022-08-08 15:23:46.99] DEBUG: update.db @ 122: Closed db
[2022-08-08 15:23:46.100] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 15:38:47.81] INFO: update.update @ 249: 2022
[2022-08-08 15:38:47.82] INFO: update.update @ 254: CAV
[2022-08-08 15:38:47.82] DEBUG: update.db @ 112: Loading db
[2022-08-08 15:38:47.83] INFO: update.update @ 258: 81
[2022-08-08 15:38:47.84] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 15:38:54.297] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf3306a0>
[2022-08-08 15:39:01.298] DEBUG: update.db @ 120: Closing db
[2022-08-08 15:39:01.298] DEBUG: update.db @ 122: Closed db
[2022-08-08 15:39:01.299] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 15:54:02.275] INFO: update.update @ 249: 2022
[2022-08-08 15:54:02.276] INFO: update.update @ 254: CAV
[2022-08-08 15:54:02.276] DEBUG: update.db @ 112: Loading db
[2022-08-08 15:54:02.278] INFO: update.update @ 258: 81
[2022-08-08 15:54:02.279] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 15:54:09.799] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaed8c9d0>
[2022-08-08 15:54:16.944] DEBUG: update.db @ 120: Closing db
[2022-08-08 15:54:16.945] DEBUG: update.db @ 122: Closed db
[2022-08-08 15:54:16.946] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 16:09:17.916] INFO: update.update @ 249: 2022
[2022-08-08 16:09:17.916] INFO: update.update @ 254: CAV
[2022-08-08 16:09:17.917] DEBUG: update.db @ 112: Loading db
[2022-08-08 16:09:17.918] INFO: update.update @ 258: 81
[2022-08-08 16:09:17.918] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 16:09:25.103] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaeebd0d0>
[2022-08-08 16:09:31.334] DEBUG: update.db @ 120: Closing db
[2022-08-08 16:09:31.334] DEBUG: update.db @ 122: Closed db
[2022-08-08 16:09:31.335] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 16:24:32.309] INFO: update.update @ 249: 2022
[2022-08-08 16:24:32.310] INFO: update.update @ 254: CAV
[2022-08-08 16:24:32.311] DEBUG: update.db @ 112: Loading db
[2022-08-08 16:24:32.313] INFO: update.update @ 258: 81
[2022-08-08 16:24:32.313] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 16:24:38.725] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdafb1bac0>
[2022-08-08 16:24:45.91] DEBUG: update.db @ 120: Closing db
[2022-08-08 16:24:45.92] DEBUG: update.db @ 122: Closed db
[2022-08-08 16:24:45.93] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 16:39:46.110] INFO: update.update @ 249: 2022
[2022-08-08 16:39:46.110] INFO: update.update @ 254: CAV
[2022-08-08 16:39:46.111] DEBUG: update.db @ 112: Loading db
[2022-08-08 16:39:46.112] INFO: update.update @ 258: 81
[2022-08-08 16:39:46.114] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 16:39:53.293] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf2925e0>
[2022-08-08 16:39:59.500] DEBUG: update.db @ 120: Closing db
[2022-08-08 16:39:59.501] DEBUG: update.db @ 122: Closed db
[2022-08-08 16:39:59.502] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 16:55:00.546] INFO: update.update @ 249: 2022
[2022-08-08 16:55:00.547] INFO: update.update @ 254: CAV
[2022-08-08 16:55:00.547] DEBUG: update.db @ 112: Loading db
[2022-08-08 16:55:00.549] INFO: update.update @ 258: 81
[2022-08-08 16:55:00.549] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 16:55:07.222] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf182fd0>
[2022-08-08 16:55:14.164] DEBUG: update.db @ 120: Closing db
[2022-08-08 16:55:14.164] DEBUG: update.db @ 122: Closed db
[2022-08-08 16:55:14.165] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 17:10:15.200] INFO: update.update @ 249: 2022
[2022-08-08 17:10:15.200] INFO: update.update @ 254: CAV
[2022-08-08 17:10:15.201] DEBUG: update.db @ 112: Loading db
[2022-08-08 17:10:15.203] INFO: update.update @ 258: 81
[2022-08-08 17:10:15.204] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 17:10:21.935] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf680d90>
[2022-08-08 17:10:28.141] DEBUG: update.db @ 120: Closing db
[2022-08-08 17:10:28.142] DEBUG: update.db @ 122: Closed db
[2022-08-08 17:10:28.142] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 17:25:29.125] INFO: update.update @ 249: 2022
[2022-08-08 17:25:29.126] INFO: update.update @ 254: CAV
[2022-08-08 17:25:29.127] DEBUG: update.db @ 112: Loading db
[2022-08-08 17:25:29.128] INFO: update.update @ 258: 81
[2022-08-08 17:25:29.129] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 17:25:36.134] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf53c880>
[2022-08-08 17:25:43.551] DEBUG: update.db @ 120: Closing db
[2022-08-08 17:25:43.551] DEBUG: update.db @ 122: Closed db
[2022-08-08 17:25:43.552] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 17:40:44.560] INFO: update.update @ 249: 2022
[2022-08-08 17:40:44.561] INFO: update.update @ 254: CAV
[2022-08-08 17:40:44.562] DEBUG: update.db @ 112: Loading db
[2022-08-08 17:40:44.563] INFO: update.update @ 258: 81
[2022-08-08 17:40:44.563] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 17:40:51.374] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf4f31f0>
[2022-08-08 17:40:57.701] DEBUG: update.db @ 120: Closing db
[2022-08-08 17:40:57.702] DEBUG: update.db @ 122: Closed db
[2022-08-08 17:40:57.702] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 17:55:58.715] INFO: update.update @ 249: 2022
[2022-08-08 17:55:58.716] INFO: update.update @ 254: CAV
[2022-08-08 17:55:58.716] DEBUG: update.db @ 112: Loading db
[2022-08-08 17:55:58.717] INFO: update.update @ 258: 81
[2022-08-08 17:55:58.717] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-08 17:56:05.872] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7fcdaf83ee80>
[2022-08-08 17:56:13.28] DEBUG: update.db @ 120: Closing db
[2022-08-08 17:56:13.28] DEBUG: update.db @ 122: Closed db
[2022-08-08 17:56:13.29] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-08 18:11:14.134] DEBUG: update.db @ 120: Closing db
[2022-08-08 18:11:14.134] DEBUG: update.db @ 122: Closed db
[2022-08-08 18:11:14.134] INFO: update.update_db @ 87: Sleeping for 49725 seconds
[2022-08-08 20:20:56.486] INFO: __init__.create_app @ 30: App startup in production mode
[2022-08-08 20:20:56.488] INFO: update.update_public_holidays @ 215: Updating public holidays
[2022-08-08 20:20:56.488] DEBUG: update.db @ 112: Loading db
[2022-08-08 20:20:58.11] DEBUG: update.update_public_holidays @ 228: Updated public holidays
[2022-08-08 20:20:58.11] DEBUG: update.public_holidays @ 196: Loading public holidays
[2022-08-08 20:20:58.15] DEBUG: update.db @ 120: Closing db
[2022-08-08 20:20:58.16] DEBUG: update.db @ 122: Closed db
[2022-08-08 20:20:58.18] INFO: update.update_db @ 87: Sleeping for 41943 seconds
[2022-08-09 00:07:21.987] INFO: __init__.create_app @ 30: App startup in production mode
[2022-08-09 00:07:21.989] INFO: update.update_public_holidays @ 215: Updating public holidays
[2022-08-09 00:07:21.989] DEBUG: update.db @ 112: Loading db
[2022-08-09 00:07:22.844] DEBUG: update.update_public_holidays @ 228: Updated public holidays
[2022-08-09 00:07:22.844] DEBUG: update.public_holidays @ 196: Loading public holidays
[2022-08-09 00:07:22.848] DEBUG: update.db @ 120: Closing db
[2022-08-09 00:07:22.848] DEBUG: update.db @ 122: Closed db
[2022-08-09 00:07:22.848] INFO: update.update_db @ 87: Sleeping for 28358 seconds
[2022-08-09 03:52:06.854] INFO: __init__.create_app @ 30: App startup in production mode
[2022-08-09 03:52:06.857] INFO: update.update_public_holidays @ 215: Updating public holidays
[2022-08-09 03:52:06.858] DEBUG: update.db @ 112: Loading db
[2022-08-09 03:52:08.136] DEBUG: update.update_public_holidays @ 228: Updated public holidays
[2022-08-09 03:52:08.136] DEBUG: update.public_holidays @ 196: Loading public holidays
[2022-08-09 03:52:08.140] DEBUG: update.db @ 120: Closing db
[2022-08-09 03:52:08.140] DEBUG: update.db @ 122: Closed db
[2022-08-09 03:52:08.140] INFO: update.update_db @ 87: Sleeping for 14873 seconds
[2022-08-09 04:56:58.244] INFO: __init__.create_app @ 30: App startup in production mode
[2022-08-09 04:56:58.245] INFO: update.update_public_holidays @ 215: Updating public holidays
[2022-08-09 04:56:58.245] DEBUG: update.db @ 112: Loading db
[2022-08-09 04:56:58.671] DEBUG: update.update_public_holidays @ 228: Updated public holidays
[2022-08-09 04:56:58.671] DEBUG: update.public_holidays @ 196: Loading public holidays
[2022-08-09 04:56:58.674] DEBUG: update.db @ 120: Closing db
[2022-08-09 04:56:58.674] DEBUG: update.db @ 122: Closed db
[2022-08-09 04:56:58.675] INFO: update.update_db @ 87: Sleeping for 10981 seconds
[2022-08-09 05:43:32.593] INFO: __init__.create_app @ 30: App startup in production mode
[2022-08-09 05:43:32.594] INFO: update.update_public_holidays @ 215: Updating public holidays
[2022-08-09 05:43:32.594] DEBUG: update.db @ 112: Loading db
[2022-08-09 05:43:34.191] DEBUG: update.update_public_holidays @ 228: Updated public holidays
[2022-08-09 05:43:34.192] DEBUG: update.public_holidays @ 196: Loading public holidays
[2022-08-09 05:43:34.195] DEBUG: update.db @ 120: Closing db
[2022-08-09 05:43:34.195] DEBUG: update.db @ 122: Closed db
[2022-08-09 05:43:34.196] INFO: update.update_db @ 87: Sleeping for 8187 seconds
[2022-08-09 08:00:09.473] INFO: update.update @ 249: 2022
[2022-08-09 08:00:09.474] INFO: update.update @ 254: CAV
[2022-08-09 08:00:09.474] DEBUG: update.db @ 112: Loading db
[2022-08-09 08:00:09.478] INFO: update.update @ 258: 81
[2022-08-09 08:00:09.479] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-09 08:00:16.775] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7f0b45e1b490>
[2022-08-09 08:00:24.453] DEBUG: update.db @ 120: Closing db
[2022-08-09 08:00:24.453] DEBUG: update.db @ 122: Closed db
[2022-08-09 08:00:24.454] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-09 08:15:25.426] INFO: update.update @ 249: 2022
[2022-08-09 08:15:25.426] INFO: update.update @ 254: CAV
[2022-08-09 08:15:25.427] DEBUG: update.db @ 112: Loading db
[2022-08-09 08:15:25.428] INFO: update.update @ 258: 81
[2022-08-09 08:15:25.428] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-09 08:15:31.574] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7f0b45c78f40>
[2022-08-09 08:15:38.299] DEBUG: update.db @ 120: Closing db
[2022-08-09 08:15:38.299] DEBUG: update.db @ 122: Closed db
[2022-08-09 08:15:38.300] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-09 08:30:39.302] INFO: update.update @ 249: 2022
[2022-08-09 08:30:39.304] INFO: update.update @ 254: CAV
[2022-08-09 08:30:39.304] DEBUG: update.db @ 112: Loading db
[2022-08-09 08:30:39.305] INFO: update.update @ 258: 81
[2022-08-09 08:30:39.306] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-09 08:30:46.199] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7f0b45a41a30>
[2022-08-09 08:30:53.527] DEBUG: update.db @ 120: Closing db
[2022-08-09 08:30:53.527] DEBUG: update.db @ 122: Closed db
[2022-08-09 08:30:53.528] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-09 08:45:54.456] INFO: update.update @ 249: 2022
[2022-08-09 08:45:54.456] INFO: update.update @ 254: CAV
[2022-08-09 08:45:54.456] DEBUG: update.db @ 112: Loading db
[2022-08-09 08:45:54.460] INFO: update.update @ 258: 81
[2022-08-09 08:45:54.460] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-09 08:46:00.492] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7f0b45884400>
[2022-08-09 08:46:06.711] DEBUG: update.db @ 120: Closing db
[2022-08-09 08:46:06.712] DEBUG: update.db @ 122: Closed db
[2022-08-09 08:46:06.713] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-09 09:01:07.582] INFO: update.update @ 249: 2022
[2022-08-09 09:01:07.582] INFO: update.update @ 254: CAV
[2022-08-09 09:01:07.582] DEBUG: update.db @ 112: Loading db
[2022-08-09 09:01:07.583] INFO: update.update @ 258: 81
[2022-08-09 09:01:07.583] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-09 09:01:13.702] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7f0b456c4310>
[2022-08-09 09:01:19.811] DEBUG: update.db @ 120: Closing db
[2022-08-09 09:01:19.811] DEBUG: update.db @ 122: Closed db
[2022-08-09 09:01:19.812] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-09 09:16:20.770] INFO: update.update @ 249: 2022
[2022-08-09 09:16:20.771] INFO: update.update @ 254: CAV
[2022-08-09 09:16:20.771] DEBUG: update.db @ 112: Loading db
[2022-08-09 09:16:20.774] INFO: update.update @ 258: 81
[2022-08-09 09:16:20.775] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-09 09:16:27.258] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7f0b454ffdc0>
[2022-08-09 09:16:34.139] DEBUG: update.db @ 120: Closing db
[2022-08-09 09:16:34.139] DEBUG: update.db @ 122: Closed db
[2022-08-09 09:16:34.140] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-09 09:31:35.81] INFO: update.update @ 249: 2022
[2022-08-09 09:31:35.83] INFO: update.update @ 254: CAV
[2022-08-09 09:31:35.83] DEBUG: update.db @ 112: Loading db
[2022-08-09 09:31:35.85] INFO: update.update @ 258: 81
[2022-08-09 09:31:35.85] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-09 09:31:44.273] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7f0b45341790>
[2022-08-09 09:31:51.505] DEBUG: update.db @ 120: Closing db
[2022-08-09 09:31:51.505] DEBUG: update.db @ 122: Closed db
[2022-08-09 09:31:51.506] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-09 09:46:52.434] INFO: update.update @ 249: 2022
[2022-08-09 09:46:52.434] INFO: update.update @ 254: CAV
[2022-08-09 09:46:52.434] DEBUG: update.db @ 112: Loading db
[2022-08-09 09:46:52.435] INFO: update.update @ 258: 81
[2022-08-09 09:46:52.436] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-09 09:46:58.838] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7f0b45eb3310>
[2022-08-09 09:47:05.642] DEBUG: update.db @ 120: Closing db
[2022-08-09 09:47:05.642] DEBUG: update.db @ 122: Closed db
[2022-08-09 09:47:05.643] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-09 10:02:06.678] INFO: update.update @ 249: 2022
[2022-08-09 10:02:06.678] INFO: update.update @ 254: CAV
[2022-08-09 10:02:06.679] DEBUG: update.db @ 112: Loading db
[2022-08-09 10:02:06.680] INFO: update.update @ 258: 81
[2022-08-09 10:02:06.681] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-09 10:02:14.435] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7f0b45f6bdf0>
[2022-08-09 10:02:21.895] DEBUG: update.db @ 120: Closing db
[2022-08-09 10:02:21.896] DEBUG: update.db @ 122: Closed db
[2022-08-09 10:02:21.896] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-09 10:17:22.859] INFO: update.update @ 249: 2022
[2022-08-09 10:17:22.859] INFO: update.update @ 254: CAV
[2022-08-09 10:17:22.859] DEBUG: update.db @ 112: Loading db
[2022-08-09 10:17:22.861] INFO: update.update @ 258: 81
[2022-08-09 10:17:22.861] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-09 10:17:30.361] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7f0b45dac760>
[2022-08-09 10:17:37.388] DEBUG: update.db @ 120: Closing db
[2022-08-09 10:17:37.388] DEBUG: update.db @ 122: Closed db
[2022-08-09 10:17:37.389] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-09 10:32:38.293] INFO: update.update @ 249: 2022
[2022-08-09 10:32:38.294] INFO: update.update @ 254: CAV
[2022-08-09 10:32:38.294] DEBUG: update.db @ 112: Loading db
[2022-08-09 10:32:38.295] INFO: update.update @ 258: 81
[2022-08-09 10:32:38.296] DEBUG: update.browser @ 129: Starting RoboBrowser session
[2022-08-09 10:32:46.480] ERROR: update.update_db @ 78: Unexpected error: current_matter=Matter(type='CAV', number=82, year=2022, title=None, deceased_name=None, flags=None): 
Traceback (most recent call last):
  File "/home/runner/probate/probatewebapp/update.py", line 71, in update_db
    updater.update(years)
  File "/home/runner/probate/probatewebapp/update.py", line 262, in update
    self.search_matter(Matter(matter_type, number, year, None, None, None))
  File "/home/runner/probate/probatewebapp/update.py", line 286, in search_matter
    search_form = self.browser.get_form()
  File "/home/runner/probate/probatewebapp/update.py", line 143, in browser
    search_form[DIVISION_SELECTOR_NAME].value = 'Probate'
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 109, in value
    self._value = self._value_to_index(value)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/robobrowser/forms/fields.py", line 99, in _value_to_index
    raise ValueError('Option {0} not found in field {1!r}'.format(value, self))
ValueError: Option Probate not found in field <robobrowser.forms.fields.Select object at 0x7f0b45cdf760>
[2022-08-09 10:32:52.762] DEBUG: update.db @ 120: Closing db
[2022-08-09 10:32:52.762] DEBUG: update.db @ 122: Closed db
[2022-08-09 10:32:52.763] INFO: update.update_db @ 87: Sleeping for 900 seconds
[2022-08-09 10:43:46.986] INFO: __init__.create_app @ 30: App startup in production mode
[2022-08-09 10:43:46.989] INFO: update.update_public_holidays @ 215: Updating public holidays
[2022-08-09 10:43:46.990] DEBUG: update.db @ 112: Loading db
[2022-08-09 10:43:48.626] DEBUG: update.update_public_holidays @ 228: Updated public holidays
[2022-08-09 10:43:48.626] DEBUG: update.public_holidays @ 196: Loading public holidays
[2022-08-09 10:43:48.630] INFO: update.update @ 249: 2022
[2022-08-09 10:43:48.631] INFO: update.update @ 254: CAV
[2022-08-09 10:43:48.639] INFO: update.update @ 258: 81
[2022-08-09 10:43:48.640] DEBUG: update.browser @ 129: Starting RoboBrowser session
